examples/CRISP/client/.env.example

```
@@ -1,7 +1,7 @@
 VITE_ENCLAVE_API=http://127.0.0.1:4000
 VITE_WALLETCONNECT_PROJECT_ID=
-# A token with a public mint function. Also used as the fee token for Enclave. 
-# This is its default address in Anvil node
-VITE_CRISP_TOKEN=0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
+# Voting eligibility token (MockVotingToken). Set after CRISP deploy — see deploy output / server .env.
+# Enclave fee token (MockUSDC) is separate: packages/enclave-contracts/deployed_contracts.json → MockUSDC.
+VITE_CRISP_TOKEN=0x9d4454B023096f34B160D6B654540c56A1F81688
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_CRISP_TOKEN key should go before the VITE_ENCLAVE_API key

(UnorderedKey)